_projects/2024/100419840/100419840.Rmd

```diff
@@ -0,0 +1,598 @@
+---
+title: "How housing is no longer a right"
+description: Visualization of the housing crisis in Spain according to official data from the Ministry of Transportation.
+categories: "2024"
+author: Maria Carda Gargallo
```

Comment:
Carda-Gargallo. Otherwise, the template would think that only Gargallo is your surname.